diff --git a/node_modules/@multiformats/multiaddr/dist/src/protocols-table.js b/node_modules/@multiformats/multiaddr/dist/src/protocols-table.js
index 1ee89f9..ed10f8e 100644
--- a/node_modules/@multiformats/multiaddr/dist/src/protocols-table.js
+++ b/node_modules/@multiformats/multiaddr/dist/src/protocols-table.js
@@ -32,6 +32,7 @@ export const table = [
     [445, 296, 'onion3'],
     [446, V, 'garlic64'],
     [460, 0, 'quic'],
+    [461, 0, 'quic-v1'],
     [465, 0, 'webtransport'],
     [466, V, 'certhash'],
     [477, 0, 'ws'],
diff --git a/node_modules/@multiformats/multiaddr/src/protocols-table.ts b/node_modules/@multiformats/multiaddr/src/protocols-table.ts
index 8d864cd..2e352ea 100644
--- a/node_modules/@multiformats/multiaddr/src/protocols-table.ts
+++ b/node_modules/@multiformats/multiaddr/src/protocols-table.ts
@@ -41,6 +41,7 @@ export const table: Array<[number, number, string, boolean?, boolean?]> = [
   [445, 296, 'onion3'],
   [446, V, 'garlic64'],
   [460, 0, 'quic'],
+  [461, 0, 'quic-v1'],
   [465, 0, 'webtransport'],
   [466, V, 'certhash'],
   [477, 0, 'ws'],
